src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/streaming_endpoint.py

```diff
@@ -64,6 +66,66 @@ def remove_akamai_access_control(client, account_name, resource_group_name, stre
     return client.update(resource_group_name, account_name, streaming_endpoint_name, streaming_endpoint)
 
 
+def update_streaming_endpoint_setter(client, resource_group_name, account_name, streaming_endpoint_name,
+                             parameters):
+    if parameters.access_control is not None:
```

Comment:
The logic from 71 to 78 will make mandatory to the CLI user to set the ips parameter (with generic updates)... Instead, I would prefer something like `if parameters.access_control is not None and <nextprop> is not None and<nextprop> is not None` to put the mapping logic inside to just doing it when the user wants to.

Reply:
I just fixed this before reading it, it wasn't passing the tests